_includes/com-ingest/com-ingest-kafka-avro-params.md

```diff
@@ -0,0 +1,10 @@
+| Parameter | Description | Required | Further information |
+|---|---|---|---|
+| `namespace` |  |  |  |
+| `type` | Valid values are `Record`, `VALUE`, `VALUE` |  |  |
+| `name` |  |  |  |
+| `doc` |  |  |  |
+| `fields` |  |  |  |
+| `name` |  |  |  |
+| `doc` |  |  |  |
+| `type` | Value Data type |  |  |
```

Comment:
This is a mistake and thanks for picking. I'm going to make changes to the com-ingest-source-kafka-avro.md file and will ping you for another review

Reply:
I've removed the include file because it's wrong.
I've added JSON syntax and arguments/parameters directly to `/docs/community/com-ingest/com-ingest-source-kafka-avro.md`
This content undoubtedly contains incorrect assumptions which I need further information on please :)

Reply:
looks good to me!

Reply:
Can you give any clarification on what to put into the table columns?